include/libtorrent/stat.hpp

```diff
@@ -75,18 +75,18 @@ namespace libtorrent
 
 		// should be called once every second
 		void second_tick(int tick_interval_ms);
-		int rate() const { return m_5_sec_average; }
-		int low_pass_rate() const { return m_5_sec_average; }
+		int rate() const { return int(m_5_sec_average); }
+		int low_pass_rate() const { return int(m_5_sec_average); }
 
-		std::int64_t total() const { return m_total_counter; }
+		std::int64_t total() const { return std::int64_t(m_total_counter); }
```

Comment:
actually.. would changing the member variables here to be signed silence the warnings you see? If so, I think I'd prefer that